ts/Extensions/ExportData.ts

```diff
@@ -1259,7 +1259,7 @@ Chart.prototype.hideData = function (): void {
 
 Chart.prototype.toggleDataTable = function (): void {
     var exportDivElements = this.exportDivElements,
-        menuItems = exportingOptions?.buttons?.contextButton.menuItems,
+        menuItems = this.options.exporting?.buttons?.contextButton.menuItems,
```

Comment:
Should we cast `this.options` to `any` @bre1470 instaed of using `.?` ?

Reply:
Or I can cast it like this, what do you think?
`menuItems = (this.options.exporting as Highcharts.ExportingOptions).buttons?.contextButton.menuItems,`

Reply:
There should be no forced casts without conditional tests. Also new code should not introduce `any` casts.

IMHO this code should be tested with `&&` instead of optional accessors. Patterns like `something?.else` are bloating the JS output because of `undefined` and `null` tests (to avoid filtering zero and empty string).